lms/static/js/learner_dashboard/views/course_enroll_view.js

```diff
@@ -29,13 +29,18 @@
                      this.$parentEl = options.$parentEl;
                      this.enrollModel = options.enrollModel;
                      this.urlModel = options.urlModel;
+                     this.grade = options.grade;
+                     this.collection_status = options.collection_status;
                      this.render();
                  },
 
                  render: function() {
-                     var filledTemplate;
+                     var filledTemplate,
+                         context = this.model.toJSON();
                      if (this.$parentEl && this.enrollModel) {
-                         filledTemplate = this.tpl(this.model.toJSON());
+                         context.grade = this.grade;
+                         context.collection_status = this.collection_status;
```

Comment:
I don't love the name `collection_status`, because it isn't the "status" of the collection.

Maybe one of the following:
```
context.course_status = this.collection_course_status;
```
or
```
context.course_status = this.course_status;
```